addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -50,6 +50,9 @@ function factory(dependencies) {
         }
 
         async refreshActivities() {
+            if (!this.hasActivities) {
```

Comment:
@alexkuhn I don't mind the fix but I'm surprised you accept a quick return on such method instead of preventing it to be called in the first place in the wrong case. What's the difference here?

Also would create a task for a test if we don't merge it now.

Reply:
> @alexkuhn I don't mind the fix but I'm surprised you accept a quick return on such method instead of preventing it to be called in the first place in the wrong case. What's the difference here?

I was thinking of not calling the method in the 1st place, but it didn't feel more natural for me while adding more code in `_updateAfter`. The fix in not super nice, but considering that it solves a traceback in very few lines of code, somewhat solid with our use of chatter (if python has no activity_ids, we surely don't want to set `hasActivities`), I'm quite happy with it for now.

> Also would create a task for a test if we don't merge it now.

Yes, we could. I was not sure it was worth the effort for the moment, considering it should be tested with a tour.

Reply:
FYI task-2316769